Apps/phjira/jira_consts.py

```diff
@@ -89,8 +89,8 @@
 JIRA_ERR_COMMENT_SET_STATUS_FAIL += "(changing the status of the ticket to Closed and then, trying to add comment to it is one such scenario)."
 JIRA_SUCC_TICKET_UPDATED = "Successfully updated the ticket"
 JIRA_SUCC_TICKET_DELETED = "Successfully deleted the ticket"
-JIRA_ERR_INPUT_FIELDS_NOT_THE_ONLY_ONE = "Invalid fields value"
-JIRA_ERR_INPUT_FIELDS_NOT_THE_ONLY_ONE += " The input json has a 'fields' key in it in addition to other keys"
+JIRA_ERR_INPUT_FIELDS_NOT_THE_ONLY_ONE = "Invalid fields value."
+JIRA_ERR_INPUT_FIELDS_NOT_THE_ONLY_ONE += " The input json has a 'fields' key in it in addition to other keys."
 JIRA_ERR_INPUT_FIELDS_NOT_THE_ONLY_ONE += " Either specify a dictionary with only one parent 'fields' key or multiple keys without the 'fields' key"
```

Comment:
this seems less readable than just having a single definition for the constant and using `\` to connect the strings on each line

Reply:
Fixed